### PR TITLE
[cifuzz] Fix timeout in reproduce.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Consider the testcase unreproducible.

Fixes: https://github.com/google/oss-fuzz/issues/6952

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
CC @DavidKorczynski

File: infra/cifuzz/fuzz_target.py [hunk 1]
```diff
@@ -247,10 +247,14 @@ def is_reproducible(self, testcase, target_path, reproduce_args):
...
-                                       testcase,
-                                       arguments=reproduce_args,
-                                       max_time=REPRODUCE_TIME_SECONDS)
+        try:
+          result = engine_impl.reproduce(target_path,
+                                         testcase,
+                                         arguments=reproduce_args,
+                                         max_time=REPRODUCE_TIME_SECONDS)
+        except TimeoutError:
+          logging.info('Reproducing with %s timed out.', target_path)
```
Comment (DavidKorczynski) [inline infra/cifuzz/fuzz_target.py:256]: Would it make sense to propagate the `TimeoutError` message from clusterfuzz https://github.com/google/clusterfuzz/blob/e9ff4b99f82cfdba0d370a06db5c882af9a94313/src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py#L390 ?

**Review by DavidKorczynski — APPROVED** [review]
LGTM, let a minor thought but am happy either or

**Review by oliverchang — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
Is this one ready for merge @jonathanmetzman ?

**Pull request merged**